extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -491,6 +491,20 @@
 		_t.buttonDown = _t.parentNode.querySelector(o.rollerblind.down);
 		_t.buttonStop = _t.parentNode.querySelector(o.rollerblind.stop);
 
+		_t.hasValue = _t.parentNode.getAttribute("data-has-value") === "true";
+		_t.valueNode = _t.parentNode.parentNode.querySelector(o.formValue);
+		_t.setValuePrivate = function(value) {
+			if (!_t.hasValue) {
+				return;
+			}
+			if (value === "DOWN") {
```

Comment:
My intention was not to remove the accepted state type.

> The problem is that such a change would break the auto update feature, because this is the one which currently sends the UP update when an UP command is sent for the item.

I missed the auto update feature. Could you point me to the implementation?
Such a feature is used for demonstration only, or is there any other use case?